bindings.c

```diff
@@ -4029,12 +4030,15 @@ static uint64_t get_reaper_start_time_in_sec(pid_t pid)
 		return 0;
 	}
 
-	return (clockticks /= ticks_per_sec);
+	res = (clockticks / ticks_per_sec);
+	return res;
 }
 
-static uint64_t get_reaper_age(pid_t pid)
+static double get_reaper_age(pid_t pid)
 {
-	uint64_t procstart, uptime, procage;
+	uint64_t uptime_ms;
+	double procstart, procage;
+	// double procstart, uptime, procage;
```

Comment:
You'd never let anyone get away with that :-)

Reply:
Oh yeah, that's so sloppy. I'm sorry. :) Will fix!

Reply:
Should be fixed. :)